public/Stations.php

```diff
@@ -82,4 +82,16 @@ public function getRelatedOnramps($id) {
   public function getDetectors($id) {
     return Detector::fetchForStation($id);
   }
+
+  /** Will take a coordinate and return the closest stationid
+   *
+   * Will take a coordinate and return the closest stationid
+   *
+   * @param $coord
+   * @return null
+   */
+  public function getStationfromCoord($coord){
+    $s = OrderedStation::fetchAll();
+    return null;
```

Comment:
Even though we're just stubbing this out, it's probably best to return the object that you've fetched, rather than a null...
Also, a best practice when stubbing things like this is to leave notes so that other people reading the code understand *why* you chose to return a null after fetching every station ever. :-)